solutions/documentation.md

```diff
@@ -43,6 +43,30 @@ Every effort should be made to keep one canonical set of documentation. When the
 
 If branches of documentation must be created, e.g. when a new major version of software is released, additional categories can be created for the old version - e.g. `docs-v1` alongside `docs-latest`. The title of this older content should contain the version number - e.g. `v1: Getting started`.
 
+#### Base level categories
+This is the list of recommended base categories for our product discourses. These may be expanded but should be discoursed in the private Staff cateogory.  
```

Comment:
A few points:

- I'm not sure about the use of "our", I think we should be careful about implying ownership over Discourses
- Some teams may be happy to discuss category changes in public, which I think should be encouraged.
- `s/discoursed/discussed` :stuck_out_tongue: 

So how about:

> This is the list of categories that we recommend any new Discourse installations start from. Teams may choose to modify their categories going forward, but such changes should be discussed beforehand with either the team (e.g. in the private "Staff" category) or the wider community (e.g. in the "General discussion" category).

Reply:
+1, let's not let them make "discoursed" a thing